lk2nd/device/dts/msm8953/msm8953-asus-zenfone3.dts

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: GPL-2.0-only
```

Comment:
```suggestion
// SPDX-License-Identifier: BSD-3-Clause
```

I'd prefer BSD-3 for new submissions since this is what upstream linux generally wants to see for dts files, and lk2nd nowdays tries to follow that a little bit